frontends/infinite-corridor/src/scss/learning-resources.scss

```diff
@@ -18,12 +18,20 @@ $cardListSpacer: 1.5em;
 }
 
 .ic-card-row-list {
-  padding-left: 0;
+  padding-left: 0pt;
+  margin-left:0pt;
+  margin-right:0pt;
```

Comment:
I'm seeing some formatting inconsistencies here, did we not get scss linting/formatting setup when we switched over to the new project setup?

Reply:
Apparently we (I, I guess) never did (`"global:scss-lint": "echo 'TODO: Set this up.'"`). Good catch.

I just created #3848 for this. I would suggest not adding it as part of this PR. I can make a separate PR against master and, if that's merged first, rebase this.